api/base/settings/local-dist.py

```diff
@@ -40,3 +40,7 @@
     'test-anon': '1/hour',
     'send-email': '2/minute',
 }
+
+# Prereg challenge data is uploaded to this project TODO: Delete when Prereg challenge ends
+PREREG_DATA_STORE_TOKEN = None
```

Comment:
These settings need to be put in `defaults.py`. Otherwise, `settings.PREREG_DATA_STORE_GUID` will result in an `AttributeError`.